packages/cozy-sharing/src/SharingBanner/hooks/useShouldDisplay.js

```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from 'react'
+
+const sharingPathnames = ['/preview']
+
+export const useShouldDisplay = location => {
+  const [shouldDisplay, setDisplay] = useState(false)
+
+  useEffect(() => {
+    setDisplay(sharingPathnames.includes(location.pathname))
+  }, [location])
+
```

Comment:
This should not be part of this lib. The lib is not the right place to know if we have to display the banner or not. It should be done in the app's land, no?

Reply:
Having harcoded path smells not good

Reply:
Well, my idea was to plug and play the Sharing Banner so yeah it can be handled in the applications themselves or in the banner. The advantage in the banner means that only one place (in terms of code) handles it instead of multiple apps. The caveat is the level of abstraction can become a black box for the consuming app (cozy notes has no idea when or why the banner will show up)

Reply:
We should not have any black box stuff. Only declarative, so we should not have this kind of behavior harcoded in the lib. Also, /preview is very specific to drive & notes. For instance Pass doesn't have this route.